index.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>판다마켓</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <header>
+    <div class="gnb wrap">
+      <a href="/" class="logo">
+        <img src="./images/Property 1=sm.png" alt="판다마켓 로고">
+      </a>
+      <a href="/login" class="btn-login Text-lg semibold">로그인</a>
```

Comment:
❗️ 수정요청
배포 사이트에서 로그인 버튼을 누르면 당연히 파일이 없으니 에러가 납니다.
다만 파일이 있어도 다른 설정을 해주지 않으시면 같은 에러가 날 확률이 높습니다~

/login의 의미는 root 폴더의 login으로 접근하라는 의미라서, 만약 login.html 파일이 root 파일에 위치해있고
다른 설정을 안해주셨으면 아래처럼 작성해주셔야 정상동작합니다~

```suggestion
      <a href="/login.html" class="btn-login Text-lg semibold">로그인</a>
```